### DISCUSSION TITLE
Application Release Notes

### DISCUSSION THREAD

**Discussion by DwayneGibbs** [dfinke/ImportExcel#1352] [Show and tell]
Within Azure DevOps, I've created a YAML pipeline with a Task that runs a PowerShell script that creates release notes of our applications. These release notes are published to our pipeline as an Excel workbook. With ImportExcel, I'm able to utilize Export-Excel to create the file(s). 

Food for thought -> ADO (Azure DevOps) has a robust API. If you're utilizing ADO, consider pairing  your pipeline with the ADO API to obtain various details from your ADO work items that would make for documentation and/or release notes. 

Then, consider piping this out to Excel. You'd be surprised with people's reactions.

**Comment by dfinke** [comment]
Would love to see the Excel sheet that gets created. Maybe the script too?

Thanks for adding this to the discussion.
